variables.tf

```diff
@@ -136,7 +136,9 @@ variable "workflows" {
       file      = optional(string, "$SHOWFILE")
     }), {})
     infracost = optional(object({
-      enabled = optional(bool, false)
+      enabled                    = optional(bool, false)
+      platform                   = optional(string, "gitlab")
+      token_environment_variable = optional(string)
```

Comment:
can we document this somehow? I don't think it would be clear to the new user, what is this used for.

Reply:
yeah, I've added readme - please check whether it's enough ☺️